src/app/helptext/task-calendar/cloudsync/cloudsync-form.ts

```diff
@@ -81,4 +81,13 @@ export default {
     enabled_placeholder: T('Enabled'),
     enabled_tooltip: T('Enable this Cloud Sync Task. Unset to disable this Cloud\
                 Sync Task without deleting it.'),
+    
+    bwlimit_placeholder: T('Bandwidth limit'),
+    bwlimit_tooltip: T('Either single bandwidth limit or bandwidth limit schedule in rclone format.<br />\
+                Example: "08:00,512 12:00,10MB 13:00,512 18:00,30MB 23:00,off".<br />\
```

Comment:
The helptext should not have any tabs on indented lines, only a single space (otherwise the extracter grabs all the spaces too)